sympy/printing/pretty/pretty_symbology.py

```diff
@@ -649,3 +647,14 @@ def center_accent(string, accent):
     firstpart = string[:midpoint]
     secondpart = string[midpoint:]
     return firstpart + accent + secondpart
+
+
+_remove_combining = dict.fromkeys(list(range(ord('\N{COMBINING GRAVE ACCENT}'), ord('\N{COMBINING LATIN SMALL LETTER X}')))
+                                + list(range(ord('\N{COMBINING LEFT HARPOON ABOVE}'), ord('\N{COMBINING ASTERISK ABOVE}'))))
```

Comment:
I would move this up so that it's with the `is_combining` function. Actually it would make sense for `is_combining` to use this dict 

If we have this structure at module level then we can use it in the `is_combining` function as well to avoid redefining the character ranges.

Reply:
> Actually it would make sense for `is_combining` to use this dict

If we use `_remove_combining`, is there any need of keeping the `is_combining()` function? Its only one reference has been replaced with the former.

Also, I'm not exactly sure how `_remove_combining` can be used inside the function.

Reply:
The `is_combining` function could just be:
```
return ord(s) in _remove_combining
```